sns/templates/notifications.json.tpl

```diff
@@ -0,0 +1,42 @@
+{
+  "Version": "2012-10-17",
+  "Id": "default_policy",
```

Comment:
Could this be more descriptive, in case someone's reading it in a different context?

Reply:
I've changed this and the sid below to something more descriptive.